src/utils/validation.js

```diff
@@ -144,14 +158,14 @@ export const customValidations = {
     return false;
   },
   rageQuitMax({ appState, values }) {
-    if (values.shares > appState.daoMember.shares) {
+    if (+values.shares > +appState.daoMember.shares) {
       return {
         name: 'shares',
         message: `Shares to Rage Quit may not exceed ${appState.daoMember.shares}.`,
       };
     }
 
-    if (values.loot > appState.daoMember.loot) {
+    if (+values.loot > +appState.daoMember.loot) {
```

Comment:
Can we use `Number(+values.loot)` for these instead of `+`. We're trying to get away from those + operators.